tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 env_list =
     check-{style,security}
-    py{39,310,311,312}{,-xdist}{,-devdeps,-cov}
+    py{39,310,311,312,313}{,-xdist}{,-devdeps,-cov}
```

Comment:
Shouldn't 39 be removed from the list given  the constrain from `pyproject.toml`?